src/utils.js

```diff
@@ -115,3 +117,13 @@ export function partition(array, n) {
     return acc;
   }, []);
 }
+
+/**
+ * Checks if a string is an UUID, according to RFC4122.
+ *
+ * @param  {String} uuid The uuid to validate.
+ * @return {Boolean}
+ */
+export function isUUID(uuid) {
```

Comment:
Ah, maybe then :) please open a Cliquet issue, since the original regexp comes from there